media/js/firefox/new/variation-scene1.js

```diff
@@ -8,12 +8,35 @@ For 'standard' /new variations, this file will append all query params
 so the correct template is rendered on /download/thanks/.
 */
 
-(function($) {
+(function() {
     'use strict';
 
-    $('.download-link').each(function(i, link) {
-        if (link.href.indexOf('download/thanks/') > -1) {
-            link.href = link.href + location.search;
+    var downloadLinks = document.getElementsByClassName('download-link');
+    var href;
+    var newQs;
+    var params = new window._SearchParams().params;
+    var prefix;
+
+    // we need to propogate 'v' or 'xv' query params to ensure the correct template is loaded on /download/thanks/
+    // only one of these keys should be present
+
+    // 'xv' denotes a 'variant' page for a specific marketing campaign
+    if (params.hasOwnProperty('xv')) {
```

Comment:
This is perhaps a little beyond the scope of this PR but may be easy enough to add as we're touching this code. Should we also add a little sanitisation to these param values? e.g. I think we'd just need to allow alpha numeric values and `-` characters (looking at what we use currently).